components/automate-ui/README.md

```diff
@@ -57,36 +57,59 @@ While it is problematic to document this information due to the maintenance burd
 
 **Package @types/jasmine: =3.3.10**
 
-Reason: `make unit` generates this error:
-```
+Reason: `make serve` and `make unit` report this error:
+
+```text
 error TS6200: Definitions of the following identifiers conflict with those in another file: ImplementationCallback, Expected, SpyObjMethodNames, CustomEqualityTester, CustomMatcherFactory, ExpectationFailed, SpecFunction, SpyObj, jasmine
+
+25 type ImplementationCallback = jasmine.ImplementationCallback;
+   ~~~~
+
+  ../node_modules/@types/jasmine/ts3.1/index.d.ts:22:1
+    22 type ImplementationCallback = jasmine.ImplementationCallback;
+       ~~~~
+    Conflicts are in this file.
```

Comment:
I really like this pattern of documenting why PINs exist in a centralized place.  We should adopt it for our go package management as well.